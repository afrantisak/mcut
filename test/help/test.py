import os, sys
if os.system("bin/recorder --help | diff test/help/test.out -"):
    sys.exit(1)
else:
    sys.exit(0)