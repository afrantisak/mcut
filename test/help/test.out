Usage: recorder <local-ip> <remote-ip> <remote-port> [options]
Allowed options:
  --help                show this help message
  --outfmt-ascii        output character data
  --outfmt-hex          output hex codes
  --outfmt-packet       output wrapped packets
  --output-file arg     output to file

